### PR TITLE
[MINOR][SS]Remove duplicate 'add' in comment of `StructuredSessionization`.

### PR DESCRIPTION
**Pull request by beliefer**
## What changes were proposed in this pull request?

`StructuredSessionization` comment contains duplicate 'add', I think it should be changed.

## How was this patch tested?

Exists UT.

### CODE REVIEW

**Review by gaborgsomogyi — APPROVED** [review]
LGTM.

**Comment by HyukjinKwon** [comment]
It's fine. but mind taking another look and see if there are some more typos to fix? I am sure there are more.

**Review by HeartSaVioR — APPROVED** [review]
LGTM

**Comment by beliefer** [comment]
> It's fine. but mind taking another look and see if there are some more typos to fix? I am sure there are more.

Thank you. I found another two typos, but I'm not sure.

File: examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala [hunk 1]
```diff
@@ -70,15 +70,15 @@ object StructuredSessionization {
...
+    // Sessionize the events. Track number of events, start and end timestamps of session,
     // and report session updates.
     val sessionUpdates = events
       .groupByKey(event => event.sessionId)
       .mapGroupsWithState[SessionInfo, SessionUpdate](GroupStateTimeout.ProcessingTimeTimeout) {
 
         case (sessionId: String, events: Iterator[Event], state: GroupState[SessionInfo]) =>
 
-          // If timed out, then remove session and send final update
+          // If timed out, remove session and send final update
```
Comment (HyukjinKwon) [inline examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: Hm, other two occurrences are fine.

I'll just get this in - let's just keep the `and` typo fix only in this PR

```diff
-    // Sessionize the events. Track number of events, start and end timestamps of session, and
+    // Sessionize the events. Track number of events, start and end timestamps of session,
```

Comment (beliefer) [reply examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: > Hm, other two occurrences are fine.
> 
> I'll just get this in - let's just keep the `and` typo fix only in this PR
> 
> ```diff
> -    // Sessionize the events. Track number of events, start and end timestamps of session, and
> +    // Sessionize the events. Track number of events, start and end timestamps of session,
> ```

Is `Sessionize` a typo?

File: examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala [hunk 2]
```diff
@@ -70,7 +70,7 @@ object StructuredSessionization {
         line.split(" ").map(word => Event(sessionId = word, timestamp))
       }
 
-    // Sessionize the events. Track number of events, start and end timestamps of session, and
+    // Sessionize the events. Track number of events, start and end time stamps of session,
```
Comment (HyukjinKwon) [inline examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: I believe timestamp is fine ..

Comment (beliefer) [reply examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: Is `Sessionize` a typo?

Comment (beliefer) [reply examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: > I believe timestamp is fine ..

Is Sessionize a typo?

Comment (srowen) [reply examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: Not a typo. Just remove 'and' here

Comment (beliefer) [reply examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredSessionization.scala]: OK. I have recovered `timestamps`.

**Comment by HyukjinKwon** [comment]
Merged to master, branch-2.4 and branch-2.3

**Comment by beliefer** [comment]
@HyukjinKwon @srowen Thanks all your help! @gaborgsomogyi @HeartSaVioR Thanks all your review.

**Pull request closed without merging**